src/mbgl/renderer/backend_scope.cpp

```diff
@@ -1,24 +1,48 @@
 #include <mbgl/gfx/backend_scope.hpp>
 #include <mbgl/gfx/renderer_backend.hpp>
-#include <mbgl/util/thread_local.hpp>
 
 #include <cassert>
 
+#undef THREAD_LOCAL_UNSUPPORTED
+#ifdef __APPLE__
+# include <TargetConditionals.h>
+# include <Availability.h>
+# if defined(__IPHONE_OS_VERSION_MIN_REQUIRED) && __IPHONE_OS_VERSION_MIN_REQUIRED < __IPHONE_9_0
+#   define THREAD_LOCAL_UNSUPPORTED = 1
```

Comment:
Could we define `THREAD_LOCAL_SUPPORTED` instead? I get confused with "negative" defines 😄

Reply:
done :)